pkgs/by-name/kr/kraken2/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub, rsync, perl, wget, gnused, findutils, libgcc, llvmPackages, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "kraken2";
+  version = "2.1.4";
+
+  src = fetchFromGitHub {
+    owner = "DerrickWood";
+    repo = pname;
+    rev = "v2.14"; # typo of current kraken2 build. version of source and software are different.
+    sha256 = "sha256-PneYibG7/u1M32f1irvBwUWyrE4gBL3evVCSj94YGQM=";
```

```suggestion
    hash = "sha256-PneYibG7/u1M32f1irvBwUWyrE4gBL3evVCSj94YGQM=";
```
Ah yes, just use `hash` everywhere